### PR TITLE
Update adding-model3.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?branch=pr-en-us-3321#dependency-injection)
Also [this URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app-xplat/adding-model?branch=pr-en-us-3321#dependency-injection) (since it's an include).

After the 20th question on what is `=>`.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis  this seems like the perfect opportunity to give a mini-lesson on DI.  I don't show the 2nd line because I share the same code (using INCLUDE) with the macOS/Linux tutorial. While I'm telling them I don't show the DB and connection string - in the next paragraph when I show the constructor - I should make that point concrete. The `MoviesController` doesn't know what the DB construction string is or even what DB system is used.

File: aspnetcore/includes/mvc-intro/adding-model3.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Open the *Startup.cs* file and examine `ConfigureServices`:
 
 [!code-csharp[Main](../../tutorials/first-mvc-app/start-mvc/sample/MvcMovie/Startup.cs?name=snippet_cs&highlight=7-8)]
 
-The highlighted code above shows the movie database context being added to the [Dependency Injection](xref:fundamentals/dependency-injection) container.
+The highlighted code above shows the movie database context being added to the [Dependency Injection](xref:fundamentals/dependency-injection) container. The line following `services.AddDbContext<MvcMovieContext>(options =>` is not shown (see your code). It specifies the database to use and the connection string. `=>` is a [lambda operator](https://docs.microsoft.com/dotnet/articles/csharp/language-reference/operators/lambda-operator).
```
Comment (tdykstra) [inline aspnetcore/includes/mvc-intro/adding-model3.md]: "see the code" doesn't help the person who is reading the tutorial for guidance without following the steps.  I would be inclined to consider breaking up the include so you can have the complete code snippet here.

Comment (ardalis) [inline aspnetcore/includes/mvc-intro/adding-model3.md]: I can't reach review.docs URLs - they say Forbidden.

Did you want me to write something up for DI here? I'd be happy to do so. Maybe separately from your PR, so I can create my own branch and PR for it?

Comment (Rick-Anderson) [inline aspnetcore/includes/mvc-intro/adding-model3.md]: Let's wait until I update this section. We'll probably want to write something like
The Database server and connection string is set in startup - but the controllers have no knowledge of the database or construction string.

**Pull request merged**